### DISCUSSION TITLE
日圓匯率跌至32年低位

### DISCUSSION THREAD

**Discussion by Floriaand** [1101-GS4719-C/Python-Community-GS4719-C#855] [General]
之前日本央行有干預匯率止貶，但效用不大，一段時間後又開始貶值。但是我之前看過在日本的台灣人說，感覺不管日本通膨多嚴重好像都比台灣好，所以新聞上報導的很嚴重，實際上如何我們也是不得而知
網址: https://www.google.com/amp/s/www.nippon.com/hk/japan-data/h01477/amp/

**Comment by peggy0329** [comment]
日本一直與國際背道而馳，日本央行仍然繼續實施大規模量化寬鬆政策，在不升息、利差不會縮小的格局下，想阻止日圓貶值是很難的。
而日本採量化寬鬆主要原因還是希望可以讓經濟復甦，由於日本疫情警戒期較其他國家長，因此人民對於外出用餐需求或旅遊需求已經壓抑很久，因此我認為日本政府透過旅遊補助、開放觀光等，確實可以讓日本經濟成長。

而為什麼日本通膨多嚴重好像都比台灣好？
我認為應該是日本企業對漲價都會格外謹慎，而且因為經濟慢慢復甦，在原物料成本變高下，雖然日本企業營運成本提高，但因為日本政府沒有升息，因此企業借款的利率依舊處在很低的狀況下，不會造成企業將成本大量的轉移到物價上，這就使日本通膨會稍微較台灣好

**Comment by williamyen0623** [comment]
好想去日本，因為貶值，東西都變得好便宜～

**Comment by Kotori0501** [comment]
感謝分享看到這篇文章總覺得好想去宅宅夢寐以求的秋葉原聖地逛街。

**Comment by Evan0979** [comment]
聽說日圓匯率的變動有影響到股價,我同學差點就虧爛哈哈哈

**Comment by CPCpyt** [comment]
不過相比日圓下跌，台幣也在下跌，
可能對於台幣來說是賺了匯差，但賠了利差。
我只能說美元最強最棒！

**Comment by orianna901001** [comment]
我原本一直在觀望日圓最低點，結果錯過之後，反而買到0.23左右的匯率，然後直到現在，現在的匯率還是上不去，看來這筆錢要卡在那很久了。

**Comment by CPW0919** [comment]
這個六月要去日本了! 好期待 感覺可以趁機多換一點日幣到時候就可以買更多東西!!

**Comment by Jamesmoneya** [comment]
真的越來越低，今天又突然大跌有望跌到21.多，希望可以抓到最低點多換一些日幣，結束期末地獄就要去日本放鬆了

**Comment by alicia0725** [comment]
因為有在關注外幣市場，剛好在很低的0.21的時候大量買入，還記得在疫情錢匯率高達0.28，目前也大概維持在0.23左右，但光當初大量買入現在賣出就可以賺很大一筆。我建議大家可以多關注各國的匯率市場！

**Comment by Brownnnn78** [comment]
最近一直在看日本匯率想趕快換錢然後把握機會去日本玩，但後來摸了摸口袋發現口袋不夠深，現在又是個大三老人沒什麼機會無壓力的出去玩，真希望畢業旅行能夠跑出國。

**Comment by juliaouo** [comment]
有規劃下一個暑假要去日本旅遊，所以其實滿想趁低點的時候換一點日幣的，但目前還沒有很多錢XD感覺手續費也是一筆不小的開銷，還是要考慮等存多點再換，很糾結。

**Comment by Alexgns** [comment]
我認為想去日本玩的話目前應該是兌換日圓不錯的價位，但若是要做投資方面，目前我不會想買入，畢竟不知道會貶值到甚麼時候，多久後才會回升
